app_feup/lib/view/theme.dart

```diff
@@ -9,135 +9,93 @@ const Color _strongGrey = Color.fromARGB(255, 90, 90, 90);
 const Color _mildBlack = Color.fromARGB(255, 0x30, 0x30, 0x30);
 
 ThemeData applicationLightTheme = ThemeData(
-    brightness: Brightness.light,
-    primaryColor: Colors.white,
-    accentColor: _darkRed,
-    dividerColor: _lightGrey,
-    hintColor: _lightGrey,
-    backgroundColor: _mildWhite,
-    scaffoldBackgroundColor: _mildWhite,
-    textTheme: TextTheme(
-      headline1: TextStyle(
-          fontSize: 72.0, fontWeight: FontWeight.bold, color: _darkRed),
-      headline2: TextStyle(
-          fontSize: 17.0, color: _mildBlack, fontWeight: FontWeight.w300),
-      headline3: TextStyle(
-          fontSize: 17.0, color: _lightRed, fontWeight: FontWeight.w300),
-      headline4: TextStyle(
-          fontSize: 17.0, color: _mildBlack, fontWeight: FontWeight.w300),
-      headline5: TextStyle(
-          fontSize: 17.0, color: _mildBlack, fontWeight: FontWeight.w400),
-      headline6: TextStyle(
-          fontSize: 17.0, color: _mildBlack, fontWeight: FontWeight.w300),
-      subtitle1:
-          TextStyle(fontSize: 17.0, color: _grey, fontWeight: FontWeight.w300),
-      subtitle2:
-          TextStyle(fontSize: 16.0, color: _grey, fontWeight: FontWeight.w300),
-      bodyText1: TextStyle(fontSize: 16.0, color: _mildBlack),
-      bodyText2: TextStyle(fontSize: 15.0, color: _mildBlack),
-      caption:
-          TextStyle(fontSize: 12.0, color: _grey, fontWeight: FontWeight.w500),
-    ),
-    iconTheme: IconThemeData(color: _strongGrey),
-    unselectedWidgetColor: _grey,
-    toggleableActiveColor: _darkRed,
-    tabBarTheme: TabBarTheme(
-      unselectedLabelColor: _strongGrey,
-      labelColor: _strongGrey,
-      labelPadding: EdgeInsets.all(0.0),
-    ),
-    canvasColor: _mildWhite,
-    cardColor: Colors.white,
-    elevatedButtonTheme: ElevatedButtonThemeData(
-      style: ElevatedButton.styleFrom(
+  brightness: Brightness.light,
+  primaryColor: _darkRed,
+  accentColor: _lightRed,
+  backgroundColor: _mildWhite,
+  scaffoldBackgroundColor: _mildWhite,
+  hintColor: _lightGrey,
+  dividerColor: _lightGrey,
+  tabBarTheme: TabBarTheme(
+    unselectedLabelColor: _strongGrey,
+    labelColor: _strongGrey,
+    labelPadding: EdgeInsets.all(0.0),
+    indicator: ShapeDecoration(
+        shape: UnderlineInputBorder(
+            borderSide: BorderSide(width: 4.0, color: _darkRed))),
+  ),
+  elevatedButtonTheme: ElevatedButtonThemeData(
+    style: ElevatedButton.styleFrom(
+        primary: _darkRed,
+        padding: const EdgeInsets.all(10.0),
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(12.0),
+        ),
+        textStyle: TextStyle(fontWeight: FontWeight.w400, fontSize: 15.0)),
+  ),
+  iconTheme: IconThemeData(color: _darkRed),
```

Comment:
What about dividing the icons colors in two categories?
The first one to regular icons used in the app
The second to the Uni logo.
This will allow to manage the colors based on the theme directly on this file.

```suggestion
  iconTheme: IconThemeData(color: _darkRed),
  accentIconTheme: IconThemeData(color: _darkRed),
```